packages/components/utils/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@contentful/f36-utils",
   "version": "4.0.0",
-  "private": true,
   "description": "Forma 36: utils React Component",
```

Comment:
Is there a reason to publish this and core?

Reply:
yes, I want to be able to test all the things in Codesandox

Reply:
In that case we might want to change the component generation template.

Reply:
I think making it publishable should be a deliberate choice, so I'd rather generate `private: true` and change it when it ready to go live.